jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  clearMocks: true,
+  collectCoverage: true,
+  coverageDirectory: "coverage",
+  testEnvironment: "jsdom",
+  moduleNameMapper: {'^.+\\.(css|less|scss)$': 'identity-obj-proxy'},
+}
```

Comment:
Не до конца настроен конфиг, показывает покрытие только тех файлов на которые написаны тесты, а не всего приложения
```suggestion
"collectCoverageFrom": [
    "src/**/*.{js,jsx,ts,tsx}",
    "!src/**/*.d.ts"
  ],
  "resolver": "jest-pnp-resolver",
  "testMatch": [
    '<rootDir>/**/*.{spec,test}.{js,jsx,ts,tsx}',
  ],
  "testEnvironment": "jsdom",
  "testURL": "http://localhost",
  "testPathIgnorePatterns": [
    "node_modules"
  ],
  "modulePathIgnorePatterns": [
    "/build/"
  ],
  "moduleNameMapper": {
    "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy"
  },
  "moduleFileExtensions": [
    "web.js",
    "js",
    "web.ts",
    "ts",
    "web.tsx",
    "tsx",
    "json",
    "web.jsx",
    "jsx",
    "node"
  ]
```